alf/utils/value_ops.py

```diff
@@ -180,6 +184,11 @@ def one_step_discounted_return(rewards, values, step_types, discounts):
                                       s=values.shape[0]))
 
     is_lasts = (step_types == StepType.LAST).to(dtype=torch.float32)
+    if is_lasts.ndim < values.ndim:
```

Comment:
suggest putting this ndim check inside expand_dim_as to simplify code

Reply:
removed.

Reply:
It's still there?

Reply:
Sorry, only removed one place.